enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/assets/ImageBuilderCommand.java

```diff
@@ -104,6 +112,12 @@ public CompletableFuture<Object> runCommand(String command, List<Object> argumen
                             break;
                         }
                     }
+                    try {
+                        int memory = getTotalMemory();
```

Comment:
Seems as a leftover, since the value is not used. Maybe log it for diagnosic purposes ?